### PR TITLE
Add option to copy bias-corrected obs value

### PR DESCRIPTION
**Pull request by svahl991**
We discovered that Met Office ODB2 reports contain a column called 'corvalue' which sometimes contains a bias-corrected observation value. This PR adds an option to the odbapi2nc import script that will allow users to import this bias-corrected value ('corvalue' column) instead of the un-corrected obs value ('obsvalue' column) if desired.

Also, while working on this enhancement, a problem was discovered with the ODB2 directory CMakeLists.txt file that was causing the ODB2 scripts to not get installed. (Not sure when this got broken since it used to work.) This is also fixed.

### CODE REVIEW

File: src/ODB2/CMakeLists.txt [hunk 1]
```diff
@@ -4,6 +4,14 @@ list( APPEND programs
   var_convert.py
   DESTINATION bin)
```
Comment (srherbener) [inline src/ODB2/CMakeLists.txt]: Should get rid of the "DESTINATION bin" keywords, since they are extraneous.

Comment (svahl991) [reply src/ODB2/CMakeLists.txt]: @srherbener , maybe I misunderstood, but when I tried removing the "DESTINATION bin" line I got an error when I executed the "ecbuild" command for the build directory. Restoring the removed line caused the error to go away. And all the other comparable "CMakeLists.txt" files in the repo also have the "DESTINATION bin" line in that place. Are you sure these words are extraneous?

Comment (srherbener) [reply src/ODB2/CMakeLists.txt]: Sorry, I meant only on the list ( APPEND programs .... ) command, you need "DESTINATION bin" on the install (...) command. Check out the CMakeLists.txt file in src/marine for example.

Comment (svahl991) [reply src/ODB2/CMakeLists.txt]: Ah. I did misunderstand. Sorry about that. I have committed the requested change now.

**Pull request merged**